packages/bbui/package.json

```diff
@@ -25,7 +25,7 @@
     "rollup-plugin-postcss": "^4.0.0",
     "rollup-plugin-svelte": "^7.1.0",
     "rollup-plugin-terser": "^7.0.2",
-    "svelte": "3.49.0"
+    "svelte": "^4.2.10"
```

Comment:
If we don't pin the version, can we guarantee we'll never accidentally update to a new patch version in one package and not another? I'd hate to accidentally start bundling 2 versions of svelte because of that (and it would increase bloat significantly). That's the main reason we explicitly pinned this before.

Reply:
I'm not 100% on how this works in a monorepo/workspace tbh. The only `yarn.lock` is in the project root , so I don't think this would be possible as long as every package has the same svelte version specified, even with the caret.

As I say though, not 100% so if you're more comfortable pinning it to an exact version I'm happy to do so.

Reply:
The team talked offline and we realised we can just specify the svelte version in the root of the monorepo and remove it from the child repos' `package.json`. Probably a bunch of other packages we can do this for too, but will punt on that for now.